concent-vm/Vagrantfile

```diff
@@ -12,24 +12,15 @@ Vagrant.configure("2") do |config|
 
     config.vm.network "private_network", ip: "172.40.2.3"
 
-    blockchain_disk       = './disk/blockchain_disk.vdi'
-    golem_blockchain_disk = './disk/golem_blockchain_disk.vdi'
+    golem_configuration_directory = './disk/golem_configuration_directory.vdi'
```

Comment:
Instead of creating a separate disk, it would be better to create a directory on the host and share it with the machine. The reason why we were using a disk was that sharing a directory is not the best idea if you have lots of data because by default it gets copied into the machine with `rsync`. But Golem's configuration is not big.

Please use [`vboxsf`](https://help.ubuntu.com/community/VirtualBox/SharedFolders).

It's fine to do this in a separate pull request.

Reply:
Ok. This will be doing in a separate pull request.

Reply:
I've created #309 for that.